check-signatures-format.py

```diff
@@ -7,12 +7,12 @@
 ok = True
 for file_name in sorted(os.listdir("_data/signed")):
     if not file_name.endswith(".yaml") or file_name[:-5] != file_name[:-5].strip():
-        print(file_name, "has invalid name")
+        print(file_name, "probably has an invalid name. Please wait for review or use a real name instead.")
```

Comment:
Actually this checks that a) the file extension is `.yaml`, not `.yml`, not empty extension; b) the file name doesn't start with or end with a space. Perhaps this check should be split unto two `if`s with messages like 'XXX file doesn't have .yaml extension' and 'XXX file name has excess spaces'.

Reply:
Ok resolved.